app/Commands/BaseCommand.php

```diff
@@ -34,23 +34,24 @@ public function handle(): int
         }
 
         try {
+            $clientOptions = [
+                'base_uri'    => 'https://api.spinupwp.app/v1/',
+                'http_errors' => false,
+                'headers'     => [
+                    'Authorization' => "Bearer {$this->apiToken()}",
+                    'Accept'        => 'application/json',
+                    'Content-Type'  => 'application/json',
+                    'User-Agent'    => 'SpinupWP/' . config('app.version'),
+                ],
+            ];
             if (!$this->spinupwp->hasApiKey()) {
-                $this->spinupwp->setApiKey($this->apiToken())->setClient();
+                $this->spinupwp->setApiKey($this->apiToken());
             }
```

Comment:
Is this needed now that we're always setting the client?

Reply:
I suppose not if the SDK only uses the api key to setup the client. I wasn't sure, but that seems to be the case.